src/styles/globalStyles.ts

```diff
@@ -472,6 +472,22 @@ export const GlobalStyle = createGlobalStyle`
       fill: ${Colors.whitePure};
     }
   }
+
+  /* Switch */
+
+  .ant-switch {
+    background-color: ${Colors.slate600};
```

Comment:
It's using `@disabled-color` in the internal implementation.

Maybe instead it should be either done by:
* replacing the `@disabled-color` to this one
* keeping the color as before
* replacing other styles as well (`:focus`'s `box-shadow` on that color)
?